pkg/sql/sem/tree/changefeed.go

```diff
@@ -10,9 +10,17 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgerror"
 )
 
+type ChangefeedLevel int
+
+const (
+	ChangefeedLevelTable    ChangefeedLevel = iota
+	ChangefeedLevelDatabase ChangefeedLevel = iota + 1
```

Comment:
nit: You can omit the type and `= iota + 1` for all entries after the first, e.g.,: https://github.com/cockroachdb/cockroach/blob/9f61a38b670e57873509568f67bb665e50cd1935/pkg/sql/sem/tree/with.go#L26-L31

nit: briefly describe the type and each value